index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls, or a <a>holder</a>
+has been designated to use, one or more particular confirmation methods. This
+binds a specific <a>claim</a> about a <a>subject</a> to one or more
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+around how a <a>verifier</a> can validate that the <a>holder</a> is bound to
+the <a>claims</a> presented in a <a>verifiable credential</a>.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method, or to use a
+different mechanism to validate whether the <a>holder</a> is bound to the
+presented <a>claims</a>, if required. Such decision can prevent impact on the
+<a>verifier</a>'s liability or lack thereof if not specified by other means
+such as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <div class="note">
+          <p>
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a>, where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>.
+          </p>
+          <p>
+A <a>verifier</a> can validate that the <a>holder</a> controls
+or has been designated the ability to use the confirmation method
+by verifying the <a>proof</a> of the <a>verifiable
+presentation</a> using the information in the confirmation method. The
+confirmation method can include the verification key, or the type of the
+confirmation method can define that the verification key is to be inferred from
+other <a>properties</a> in the <a>verifiable credential</a>, such as the
+<code>credentialSubject.id</code>.
+          </p>
+        </div>
+
+        <dl>
+          <dt><dfn>confirmationMethod</dfn></dt>
+          <dd>
+          <dd>
+            <p>
+If present, the value of the <code>confirmationMethod</code> <a>property</a> is
+one or more confirmation methods, each providing enough information for a
+<a>verifier</a> to validate that the <a>holder</a> generating a <a>verifiable
+presentation</a> has proven control of a confirmation method bound to one or
+more <a>claims</a> in a <a>verifiable credential</a> in the
+<a>verifiable presentation</a>. It is required that the <a>issuer</a> verifies
```

Comment:
I would like this sentence to be shorted if poss.

Possibly?
```suggestion
If present, the value of the <code>confirmationMethod</code> <a>property</a> is
one or more confirmation methods, each providing enough information for a
<a>verifier</a> to validate that the <a>holder</a> generating a <a>verifiable
presentation</a> can satisfy the confirmation method bound to one or
more of the presented <a>claims</a>. It is required that the <a>issuer</a> verifies
```

Or perhaps this?

```suggestion
If present, the value of the <code>confirmationMethod</code> <a>property</a> is
one or more confirmation methods. Each confirmation method is bound to one or more claims in the <a>verifiable credential</a> and provides enough information for a
<a>verifier</a> to validate whether the <a>holder</a> generating a <a>verifiable
presentation</a> can satisfy them. It is required that the <a>issuer</a> verifies
```

Reply:
@wip-abramson — Better to make alternative suggestions in distinct comments, as any can then be merged. GitHub doesn't have a way to merge one-of-multiple suggestions made in one comment.

Here's my version of your latter suggestion:
```suggestion
If present, the value of the <code>confirmationMethod</code> <a>property</a> is
one or more confirmation methods. Each confirmation method is bound to one or more
claims in the <a>verifiable credential</a>, and provides enough information for a
<a>verifier</a> to confirm whether the <a>holder</a> generating a <a>verifiable
presentation</a> can satisfy that confirmation method. It is required that the
<a>issuer</a> verifies
```